lib/beaker/cli.rb

```diff
@@ -1,7 +1,7 @@
 module Beaker
   class CLI
     VERSION_STRING =
-"      wWWWw
+      "      wWWWw
```

Comment:
Looks like this is supposed to be some ASCII art:
```
      wWWWw
      |o o|
      | O |  %s!
      |(")|
     / \X/ \
    |   V   |
    |   |   |
```

A heredoc string without special char would help:

```
    VERSION_STRING = <<-'ART'
      wWWWw
      |o o|
      | O |  %s!
      |(")|
     / \X/ \
    |   V   |
    |   |   |
    ART
```